pyrokinetics/pyroscan.py

```diff
@@ -267,28 +267,35 @@ def load_gk_output(self):
 
             # Load gk_output in copies of pyro
             for pyro in self.pyro_dict.values():
-                pyro.load_gk_output()
-
-                growth_rate.append(pyro.gk_output.data["growth_rate"].isel(time=-1))
-                mode_frequency.append(
-                    pyro.gk_output.data["mode_frequency"].isel(time=-1)
-                )
-                eigenfunctions.append(
-                    pyro.gk_output.data["eigenfunctions"]
-                    .isel(time=-1)
-                    .drop_vars(["time"])
-                )
-                fluxes.append(
-                    pyro.gk_output.data["fluxes"]
-                    .isel(time=-1)
-                    .sum(dim="ky")
-                    .drop_vars(["time"])
-                )
-
-                pyro.gk_code.get_growth_rate_tolerance(pyro, time_range=0.9)
-                growth_rate_tolerance.append(
-                    pyro.gk_output.data["growth_rate_tolerance"]
-                )
+                try:
+                    pyro.load_gk_output()
+
+                    growth_rate.append(pyro.gk_output.data["growth_rate"].isel(time=-1))
+                    mode_frequency.append(
+                        pyro.gk_output.data["mode_frequency"].isel(time=-1)
+                    )
+                    eigenfunctions.append(
+                        pyro.gk_output.data["eigenfunctions"]
+                        .isel(time=-1)
+                        .drop_vars(["time"])
+                    )
+                    fluxes.append(
+                        pyro.gk_output.data["fluxes"]
+                        .isel(time=-1)
+                        .sum(dim="ky")
+                        .drop_vars(["time"])
+                    )
+
+                    pyro.gk_code.get_growth_rate_tolerance(pyro, time_range=0.95)
+                    growth_rate_tolerance.append(
+                        pyro.gk_output.data["growth_rate_tolerance"]
+                    )
+                except (FileNotFoundError, OSError) as e:
```

Comment:
Flake8 is complaining that `e` isn't used. Looks like there might a bug in the action as it cuts off the message.

```suggestion
                except (FileNotFoundError, OSError):
```